charts/prometheus-kafka-exporter/values.yaml

```diff
@@ -16,6 +16,10 @@ image:
   tag: latest
   pullPolicy: IfNotPresent
 
+# If enabled Kafka dependency chart will be used. Should be disabled in production environments.
+kafka:
+  enabled: true
```

Comment:
This should be false by default.
It should be enabled only through ci like in my example.